src/ripple/app/tx/applySteps.h

```diff
@@ -114,6 +115,7 @@ struct PreclaimResult
     {
     }
 
+    PreclaimResult(PreclaimResult const&) = default;
```

Comment:
@HowardHinnant This is probably out of scope for this PR, but my intention with `delete`ing the copy operator was to force moves whenever possible. Unfortunately `boost::optional` requires a copy constructor to be available. I can never remember the rules for this - are move constructors / copy operators auto-generated in the current set up? If not, would you be willing to add a couple of `default`s?

Reply:
The explicitly declared copy members will inhibit the compiler declaration and definition of the move members.  To get them you'll have to at least explicitly default them.  Though with the reference data members, no sense trying to default the move assignment operator as it will get deleted anyway.